boards/adi/max78002evkit/doc/index.rst

```diff
@@ -0,0 +1,297 @@
+.. _max78002_evkit:
+
+MAX78002EVKIT
+#############
+
+Overview
+********
+The MAX78002 evaluation kit (EV kit) provides a platform and tools for leveraging device capabilities to build new
+generations of artificial intelligence (AI) products. The kit provides optimal versatility with a modular peripheral
+architecture, allowing a variety of input and output devices to be remotely located. DVP and CSI cameras,
+I2S audio peripherals, digital microphones, and analog sensors are supported, while a pair of industry-standard
+QWIIC connectors supports a large and growing array of aftermarket development boards. An onboard stereo
+audio codec offers line-level audio input and output, and tactile input is provided by a touch-enabled 2.4in TFT
+display. The MAX78002 energy consumption is tracked by a power accumulator, with four channels of formatted
+results presented on a secondary TFT display. All device GPIOs are accessible on 0.1in pin headers. A standard
+coaxial power jack serves as power input, using the included 5V, 3A wall-mount adapter. Two USB connectors
+provide serial access to the MAX78002, one directly and the other through a USB to UART bridge. A third
+USB connector allows access to the MAX78002 energy consumption data. Rounding out the features, a microSD
```

Comment:
this is a mouthful (but not your fault as it looks copy-pasted :))
please try to add a bit of breathing room by at least adding a blank line after first sentence (just as in https://www.analog.com/en/resources/evaluation-hardware-and-software/evaluation-boards-kits/max78002evkit.html#eb-overview, basically)

Reply:
Updated according to your suggestion, thanks.